src/freenet/node/NewPacketFormat.java

```diff
@@ -992,7 +992,7 @@ public void checkForLostPackets() {
 	 * queued at plus the lesser of half the RTT or 100ms if there are acks queued. 
 	 * Otherwise Long.MAX_VALUE to indicate that we need to get messages from the queue. */
 	@Override
-	public long timeNextUrgent(boolean canSend) {
+	public long timeNextUrgent(boolean canSend, long now) {
 		long ret = Long.MAX_VALUE;
```

Comment:
This is true. However I can't imagine a plugin using NewPacketFormat. If it did it would mean it was doing *really* low level stuff e.g. transport plugins - and there isn't really usable infrastructure for that. Maybe forcing a packet send, maybe some ambitious low-level diagnostics...?